### ISSUE TITLE
Xcode Plugin integration

### ISSUE THREAD

**Issue by cesar-oyarzun-m** [facebook/infer#915]
Hi, I'm following the getting started documentation and everything works perfect from command line, I'm trying to finde some direct integration with Xcode IDE, so I can see the report there, Do you have any integration tool or some way to integrate infer with Xcode?

**Comment by dulmarod** [comment]
No, we don't have XCode plugin integration at the moment. One reason is that Infer is a bit slow on larger projects, so running quickly while editing a file is not very feasible.
